drone_localization/api/endpoints/localize.py

```
@@ -19,32 +22,58 @@
         )
         logger.info("2nd model's work is ended")
+        
+        # Получаем метаданные спутникового снимка (tl_E, tl_N, br_E, br_N)
+        if satellite_result.metadata is None:
+            raise HTTPException(status_code=500, detail="Satellite metadata not found")
+
+        logger.info("Satellit's metadata was found")
+        # Определяем размеры спутникового изображения
+        sat_width, sat_height = satellite_result.image.size
+
+        logger.info("Satellite's width and height is known:", sat_width, sat_height)
```

Ruff (0.15.12)

[error] Too many arguments for `logging` format string

(PLE1205)